src/App.vue

```diff
@@ -1,10 +1,129 @@
-<script setup>
+<script>
+export default {
+  data() {
+    return {
+      user: {
+        name: 'Anubha Rajput',
+        address: '123 Main Street',
+        dob: '2000-10-21'
+      },
+      age: null,
+      showAge: false,
+      isError: null,
+      isValid:false,
+      users: [
+        { name: 'Rahul', age: 24, imageUrl: "https://mir-s3-cdn-cf.behance.net/project_modules/max_1200/35af6a41332353.57a1ce913e889.jpg" },
+        { name: 'Rohit', age: 20, imageUrl: "https://s3-eu-west-1.amazonaws.com/files2.fd.nl/Erwin/Slider+Daan/stefan-bron-2.jpg" },
+        { name: 'Anubha', age: 21, imageUrl: "https://static.nieuwsblad.be/Assets/Images_Upload/2012/10/07/Stefan-004-kl.jpg" },
+        { name: 'amit', age: 21, imageUrl: "https://media.licdn.com/dms/image/C4E03AQGAHt1PVFJO6A/profile-displayphoto-shrink_800_800/0/1623249411702?e=2147483647&v=beta&t=PhjGA8rK15XMw5cFPTLz96KWHaYP_beHpHfqgoHj7bM" }
+      ],
+      validated: true,
+    };
+  },
+  methods: {
+    calculateAge() {
+      const today = new Date();
+      const birthDate = new Date(this.user.dob);
+      const diff = today - birthDate;
+      const ageDate = new Date(diff);
+      this.age = Math.abs(ageDate.getUTCFullYear() - 1970);
+      this.showAge=null;
+    },
+    validateAge() {
+      console.log("ok")
+      this.showAge = true;
+      if (this.age !== null) {
+        if (this.age < 18) {
+          this.isValid = false;
+          this.isError = true;
+        } else {
+          this.isValid= true;
+          this.isError = false;
+        }
+      } else {
+        this.showAge = "Please calculate age first.";
+      }
+    }
+  },
+  mounted() {
+    console.log(`The initial Age is ${this.age}.`)
+  }
+}
 </script>
-
 <template>
-  <h1>Hellol</h1>
+  <div class="main-container">
+    <p class="user-data"><span>Name:</span> {{ user.name }}</p>
```

Comment:
any reason to add span? if you are not going to add any style, @anubharajput

Reply:
in Every span tag i applied some css properties like font-size and font-family